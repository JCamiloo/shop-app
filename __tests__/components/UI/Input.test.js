import React from "react";
import Input from "../../../components/UI/Input";
import renderer from "react-test-renderer";

describe("Input component tests", () => {
  test("Input component renders correctly", () => {
    const tree = renderer.create(<Input />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
