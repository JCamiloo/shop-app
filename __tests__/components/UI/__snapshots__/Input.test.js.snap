// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input component tests Input component renders correctly 1`] = `
<View
  style={
    Object {
      "width": "100%",
    }
  }
>
  <Text
    style={
      Object {
        "fontFamily": "open-sans-bold",
        "marginVertical": 8,
      }
    }
  />
  <TextInput
    allowFontScaling={true}
    onBlur={[Function]}
    onChangeText={[Function]}
    rejectResponderTermination={true}
    style={
      Object {
        "borderBottomColor": "#ccc",
        "borderBottomWidth": 1,
        "paddingHorizontal": 2,
        "paddingVertical": 5,
      }
    }
    underlineColorAndroid="transparent"
    value=""
  />
</View>
`;
